pennylane_lightning/lightning_kokkos/lightning_kokkos.toml

```
@@ -1,124 +1,105 @@
-schema = 1
-
-[device]
-name = "lightning.kokkos"
-
-[operators]
-# Observables supported by the device
-observables = [
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "Hadamard",
-        "Hermitian",
-        "Identity",
-        "SparseHamiltonian",
-        "Hamiltonian",
-        "Sum",
-        "SProd",
-        "Prod",
-        "Exp",
-]
+schema = 2
 
 # The union of all gate types listed in this section must match what
 # the device considers "supported" through PennyLane's device API.
-[[operators.gates]]
-native = [
-        "Identity",
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "Hadamard",
-        "S",
-        "T",
-        "PhaseShift",
-        "RX",
-        "RY",
-        "RZ",
-        "Rot",
-        "CNOT",
-        "CY",
-        "CZ",
-        "SWAP",
-        "CSWAP",
-        "Toffoli",
-        "IsingXX",
-        "IsingXY",
-        "IsingYY",
-        "IsingZZ",
-        "ControlledPhaseShift",
-        "CRX",
-        "CRY",
-        "CRZ",
-        "CRot",
-        "SingleExcitation",
-        "SingleExcitationPlus",
-        "SingleExcitationMinus",
-        "DoubleExcitation",
-        "DoubleExcitationPlus",
-        "DoubleExcitationMinus",
-        "MultiRZ",
-        "QubitUnitary",
-        "GlobalPhase",
-]
+[operators.gates.native]
+
+CNOT                   = { properties = [ "invertible",                 "differentiable" ] }
+ControlledPhaseShift   = { properties = [ "invertible",                 "differentiable" ] }
+ControlledQubitUnitary = { properties = [ "invertible",                                  ] }
+CRot                   = { properties = [ "invertible"                                   ] }
+CRX                    = { properties = [ "invertible",                 "differentiable" ] }
+CRY                    = { properties = [ "invertible",                 "differentiable" ] }
+CRZ                    = { properties = [ "invertible",                 "differentiable" ] }
+CSWAP                  = { properties = [ "invertible",                 "differentiable" ] }
+CY                     = { properties = [ "invertible",                 "differentiable" ] }
+CZ                     = { properties = [ "invertible",                 "differentiable" ] }
+DoubleExcitationMinus  = { properties = [ "invertible",                 "differentiable" ] }
+DoubleExcitationPlus   = { properties = [ "invertible",                 "differentiable" ] }
+DoubleExcitation       = { properties = [ "invertible",                 "differentiable" ] }
+GlobalPhase            = { properties = [ "invertible", "differentiable" ] }
```

Minor, but since you have this nice formatting in place:
```suggestion
GlobalPhase            = { properties = [ "invertible",                 "differentiable" ] }
```